Vagrantfile

```diff
@@ -67,7 +68,16 @@ else
     # cleanup openstack-kilo repo if required
     yum remove docker-engine -y || :
     yum-config-manager --disable openstack-kilo
-    curl https://get.docker.com | sed s/docker-engine/docker-engine-#{docker_version}/ | bash
+    if [[ #{docker_version} == *"rc"* ]]; then
```

Comment:
Since we are working with pre-released docker versions as well, we need the rc version. cs is the customer support version which was suggested by docker folks.